src/app/content/hooks/locationChange/resolveContent.ts

```diff
@@ -206,7 +206,12 @@ const loadContentReference = async(
   services: AppServices & MiddlewareAPI,
   book: Book,
   page: ArchivePage,
-  reference: ReturnType<typeof getContentPageReferences>[0]
+  reference: {
```

Comment:
Can you leave this the way it was? Shouldn't it just pick up the new return type?

Reply:
it doesn’t catch `reference.bookId` and `reference.bookVersion`

Reply:
it doesn’t catch `reference.bookId` and `reference.bookVersion`